### PR TITLE
Create python package - mostly documentation updates

### PR DESCRIPTION
**Pull request by vicwomg**
Documentation and minor tweaks to make the PyPi package work. I want to update the docs so that if I merge the branch to master, the README has valid installation instructions.

- Removed outdated documentation / moved some stuff around
- Updated README to include ffmpeg/chrome installation. Remove venv install steps (probably best to not recommend it for python-inexperienced)
- Git ignore `dist` package directory
- Lower python requirement to 3.9 (default install on Raspberry Pi)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -21,49 +21,61 @@ If you want to support this project with a little monetary tip, it's much apprec
...
-| Raspberry Pi 3 High-Res Video Playback | Pi 3 might struggle with high-res video playback, but overclocking can help.                  |
-| Disable Screen Blanking               | Disable "screen blanking" in `raspi-config` to prevent the display from turning off when idle.|
-| Modern Mac, PCs, and Linux            | Works fine on modern devices.                                                                 |
+- Raspberry Pi
+  - Requires a Raspberry Pi Model 3 or higher
+  - Desktop OS required for standalone/headed mode
+  - For Pi 3: 32-bit Bullseye OS and overclocking is recommended for smoother playback.
+- OSX
+- Windows
+- Linux
```
Comment (mariugul) [inline docs/README.md:46]: Would it be better to add (experimental) behind these for the time being?

Comment (vicwomg) [reply docs/README.md:46]: I am as confident in these working as I am with raspberry pi (perhaps more so, given the eccentricities of pi hardware)

File: .gitignore [hunk 2]
```diff
@@ -3,6 +3,7 @@
 .vscode/
 .venv/
 __pycache__/
+dist/
```
Comment (vicwomg) [inline .gitignore:6]: This is the poetry build directory

File: pyproject.toml [hunk 3]
```diff
@@ -2,10 +2,10 @@
 name = "pikaraoke"
 version = "1.0.0"
 description = "Youtube-based Karaoke machine for Raspberry Pi, OSX, Windows, and Linux"
-authors = ["Marius <my_email@example.com>"]
+authors = ["Vic Wong"]
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.9"
```
Comment (vicwomg) [inline pyproject.toml:8]: Raspberry Pi Buster comes with 3.7.2. Starting with Bullseye: 3.9. Bookworm (current): 3.11.

**Pull request merged**